### PR TITLE
Reapply batch processes more than 10 objects.

### PR DESCRIPTION
**Pull request by jordandukart**
**JIRA Ticket**: https://jira.duraspace.org/browse/ISLANDORA-1908

# What does this Pull Request do?
Reapply scholar embargo batches now process more than 10 objects before exiting.

# What's new?
Batch operates correctly.

# How should this be tested?
Go to admin/islandora/solution_pack_config/embargo/roles
Reapply embargoes on a site that has > 10 objects embargoed.
Note that only 10 of the n objects get reapplied.
Pull code, note that all objects are reapplied.

# Additional Notes:
There likely needs to be some communication / announcement / light brought up to this issue as it could leave the repository in an unexpected state in the event a role was added and this batch ran previously. 

# Interested parties
@Islandora/7-x-1-x-committers, @bryjbrown, @DiegoPino

### CODE REVIEW

**Comment by bryjbrown** [comment]
@jordandukart Just tested this on a VM and indeed, batches on 7.x stop at 10 objects. Surprise, surprise! Switching to this PR makes it reindex all embargoed objects, as advertised. 

@DiegoPino, if you approve of the code then we can merge.

**Comment by DiegoPino** [comment]
@bryjbrown you both know better, feel free to approve and merge if the code is fine for you (and travis is happy) 😃 Just give it 24 hrs time (per committers guidelines suggestions) so other @Islandora/7-x-1-x-committers can comment if needed. Thanks!

**Comment by willtp87** [comment]
This seems in order to me.

**Review by bryjbrown — APPROVED** [review]
👍

**Comment by DiegoPino** [comment]
@jordandukart @bryjbrown @willtp87 thanks!

**Pull request merged**
